src/main/java/eu/dissco/backend/repository/OrganisationRepository.java

```diff
@@ -1,11 +1,13 @@
 package eu.dissco.backend.repository;
 
 import static eu.dissco.backend.database.jooq.Tables.ORGANISATION_DO;
+import static eu.dissco.backend.repository.RepositoryUtils.getOffset;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
 import eu.dissco.backend.domain.Country;
 import eu.dissco.backend.domain.OrganisationTuple;
+
 import java.util.List;
+
```

Comment:
The formatting looks off. Might be due to new Intelij. Did you import the code formatting scheme?